test/include/integration/end_to_end_test.h

```diff
@@ -0,0 +1,73 @@
+#pragma once
+
+#include <memory>
+#include <string>
+#include <utility>
+#include <vector>
+
+#include "gtest/gtest.h"
+#include "binder/bind_node_visitor.h"
+#include "execution/compiler/compilation_context.h"
+#include "optimizer/cost_model/abstract_cost_model.h"
+#include "optimizer/cost_model/trivial_cost_model.h"
+#include "parser/postgresparser.h"
+#include "traffic_cop/traffic_cop_util.h"
+#include "execution/compiler/output_checker.h"
+#include "execution/sql_test.h"
+
+namespace noisepage::test {
```

Comment:
Let me know if you have any thoughts on the placement of this file

Reply:
Personally, don't have a strong opinion, why not test_util though?

Reply:
I actually think test_util looks better, I probably just didn't see it

Reply:
Done